src/core/build/createWebpackConfig.ts

```diff
@@ -363,6 +382,22 @@ export default function createWebpackConfig({
     },
   };
 
+  const devServerEntries = isProduction
+    ? []
+    : [
+        // Include an alternative client for WebpackDevServer. A client's job is to
+        // connect to WebpackDevServer by a socket and get notified about changes.
+        // When you save a file, the client will either apply hot updates (in case
+        // of CSS changes), or refresh the page (in case of JS changes). When you
+        // make a syntax error, this client will display a syntax error overlay.
+        // Note: instead of the default WebpackDevServer client, we use a custom one
+        // to bring better experience for Create React App users. You can replace
+        // the line below with these two lines if you prefer the stock client:
+        // require.resolve('webpack-dev-server/client') + '?/',
+        // require.resolve('webpack/hot/dev-server'),
+        require.resolve("react-dev-utils/webpackHotDevClient"),
+      ];
+
```

Comment:
How is this related to cross browser support?

Reply:
Just deduplicated some lines there ;-)

Reply:
Nothing has changed